database/scripts/cambiar_tipo_dato_valor_integer_to_numeric.down.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE plan_adquisiciones."Registro_plan_adquisiciones-Actividad"
+ALTER COLUMN valor TYPE integer;
+
+ALTER TABLE plan_adquisiciones."Registro_inversion_actividad-Fuente_financiamiento"
+ALTER COLUMN valor_asignado TYPE integer;
```

Comment:
(En general) Dejar vacía la última línea de cualquier archivo de texto seguido por Git
```suggestion
ALTER COLUMN valor_asignado TYPE integer;

```